### ISSUE TITLE
Training  GPU memory fluctuation

### ISSUE THREAD

**Issue by 1goodone** [yucornetto/MGMatting#17]
When I train, I notice GPU memory fluctuations. Do you have such problems。

**Comment by yucornetto** [comment]
That does not occur to me. Will it cause any problem of training or unstable results?

**Comment by 1goodone** [comment]
No, it just feels like it's affecting the speed and I was wondering if that's the case with you

> That does not occur to me. Will it cause any problem of training or unstable results?
